VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionResponsesFragment.java

```diff
@@ -136,9 +138,15 @@ public void onEventMainThread(DiscussionCommentPostedEvent event) {
         if (discussionThread.containsComment(event.getComment())) {
             if (event.getParent() == null) {
                 // We got a response
-                courseDiscussionResponsesAdapter.addNewResponse(event.getComment());
+                ((BaseFragmentActivity)getActivity()).showInfoMessage(getString(R.string.discussion_response_posted));
+                if (!responsesLoader.hasMorePages()) {
+                    courseDiscussionResponsesAdapter.addNewResponse(event.getComment());
+                    discussionResponsesRecyclerView.smoothScrollToPosition(
+                            courseDiscussionResponsesAdapter.getItemCount() - 1);
+                }
             } else {
                 // We got a comment to a response
+                ((BaseFragmentActivity)getActivity()).showInfoMessage(getString(R.string.discussion_comment_posted));
```

Comment:
I think we don't need to show the new comments posting notification in this screen, as it's not directly interacting with it, and we aren't currently showing it globally.